templates/caddy.conf

```diff
@@ -27,5 +28,5 @@ limit nofile 1048576 1048576
 script
         cd /etc/caddy
```

Comment:
I am not sure why these scripts cd into /etc/caddy: is it supposed to be CADDYPATH (i.e. dir where certs are) or dir where Caddyfile is?